app/views/fileChecksResults.scala.html

```diff
@@ -8,7 +8,7 @@
         @progressIndicator(Messages("fileChecksResults.progress"))
         <h1 class="govuk-heading-l">Results of your checks</h1>
         <div id="upload-progress-success" class="success-summary" aria-labelledby="success-summary-title"
-        role="alert" data-module="success-summary" tabindex="-1">
+        role="status" data-module="success-summary" tabindex="-1">
```

Comment:
What's the reason for this change? It's not obvious from the commit message.

Reply:
I thought I read somewhere that alert was for warnings but I can't find it anymore, so I'll revert it to the `alert` role

Reply:
This looks like it's gone back to "status". Which might be fine! I just couldn't figure out the reason for the change from the commit message.